float64[42] zero_jacobian
